app/misc/git.ts

```diff
@@ -39,7 +39,7 @@ function sortedListOfCommits(commits){
 
     while (commits.length > 0) {
       let commit = commits.shift();
-      let parents = commit.parents();
+      let parents = commit.parentxs();
```

Comment:
Hi @hdauvic, would you mind elaborating on the change here? Was the addition of the 'x' a typo or is it a call in typescript? I couldn't find any reference and definitions other than here so I thought I'd ask for clarification. Thanks!

Reply:
I'm seeing these x's pop up from when I merged the master to update mine. I didn't add this x, nor another that I randomly found in the code. The only code I touched was anything with "console.log" in front of it. This is worrysome.

Reply:
I looked at the current master and have removed this x.